src/test/components/__snapshots__/SampleTooltipContents.test.js.snap

```diff
@@ -1,601 +1,618 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`SampleTooltipContents renders the sample tooltip properly 1`] = `
-<body>
-  <div>
+<div
```

Comment:
It would be a lot easier to see the changes if you check "Hide whitespace changes" here. Indentation changes are making them hard to see.